xchange-independentreserve/src/main/java/org/knowm/xchange/independentreserve/IndependentReserveAuthenticated.java

```diff
@@ -34,10 +31,11 @@
 @Produces(MediaType.APPLICATION_JSON)
 public interface IndependentReserveAuthenticated {
 
-  public static final String SynchDigitalCurrencyDepositAddressWithBlockchain =
+  String SynchDigitalCurrencyDepositAddressWithBlockchain =
       "SynchDigitalCurrencyDepositAddressWithBlockchain";
-  public static final String WithdrawDigitalCurrency = "WithdrawDigitalCurrency";
-  public static final String GetDigitalCurrencyDepositAddress = "GetDigitalCurrencyDepositAddress";
+  String WithdrawDigitalCurrency = "WithdrawDigitalCurrency";
+  String GetDigitalCurrencyDepositAddress = "GetDigitalCurrencyDepositAddress";
+  String GetBrokerageFees = "GetBrokerageFees";
```

Comment:
why are we getting rid of static final?

Reply:
It is an interface, static final is redundant